src/python/zquantum/core/estimation/_estimation.py

```diff
@@ -298,13 +302,13 @@ def estimate_expectation_values_by_averaging(
 
     (
         estimation_tasks_to_measure,
-        estimation_tasks_for_constants,
+        estimation_tasks_not_measured,
         indices_to_measure,
-        indices_for_constants,
-    ) = split_constant_estimation_tasks(estimation_tasks)
+        indices_not_to_measure,
+    ) = split_estimation_tasks_to_measure(estimation_tasks)
 
-    expectation_values_for_constants = evaluate_constant_estimation_tasks(
-        estimation_tasks_for_constants
+    expectation_values_for_constants = evaluate_non_measured_estimation_tasks(
```

Comment:
I'd suggest renaming `expectation_values_for_constants` to `estimation_tasks_not_measured` for clarity

Reply:
Fixed in 7271c6240983bb6beaef7c480e7deb773e8ad567